src/Appwrite/Event/Func.php

```diff
@@ -200,6 +223,7 @@ public function trigger(): string|bool
             'project' => $this->project,
             'user' => $this->user,
             'function' => $this->function,
+            'functionId' => $this->functionId,
```

Comment:
There is already $function, why cant we get function ID as `function->getId()`?

Reply:
The `resource` of the scheduler is an `execution`, not a `function`. We aren't able to pass the whole `function` resource as we only have the linked `functionId` from the `execution`.